CMake/smf.cmake

```diff
@@ -1,33 +1,54 @@
 #  1. over time this should become a proper cmake module
 
 function(smfc_gen)
-  set(options CPP GOLANG)
+  set(options CPP GOLANG VERBOSE)
   set(oneValueArgs TARGET_NAME OUTPUT_DIRECTORY)
-  set(multiValueArgs SOURCES)
+  set(multiValueArgs SOURCES INCLUDE_DIRS)
   cmake_parse_arguments(SMFC_GEN "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
+  # We need one per generator so we can pass it to the flatc compiler
+  #
+  set(flatc_generated_includes)
```

Comment:
@noahdesu  - would be cool to generate a binary dep i nstead, so you can depend on it correctly without hacking the path around. 

Do you know howto? in cmake